scripts/actions/remove-completed-batch.js

```diff
@@ -51,14 +51,14 @@ const removePageContext = async (fileUris) => {
     accessToken,
   });
 
+  // Find the smartling context to be removed. This includes context manually
+  // uploaded (fileNames) as well as the "context" created by smartling (fileUris).
   const contextUids = items
-    .filter((item) => fileNames.includes(item.name))
-    .map((item) => {
-      return {
-        fileUri: item.name,
-        contextUid: item.contextUid,
-      };
-    });
+    .filter(({ name }) => [...fileNames, ...fileUris].includes(name))
+    .map(({ name, contextUid }) => ({
+      fileUri: name,
+      contextUid: contextUid,
```

Comment:
```suggestion
      contextUid,
```
[nit] can use shorthand here

Reply:
Oh duh...I had turned this into an implicit return for the purpose of removing that 🤦‍♂️

Reply:
Haha no problem 😄